src/App.js

```diff
@@ -0,0 +1,204 @@
+import "./styles/main.scss";
+import Header from "./components/Header";
+import AddNewInput from "./components/AddNewInput";
+import Filter from "./components/FilterSortItems";
+import CheckListItem from "./components/CheckListItem";
+import React, { Component } from "react";
+import http from "./utils/http";
+import * as todoService from "./services/todolist";
+const FILTER_MAP = { 
+  All: () => true,
+  Active: (task) => !task.completed,
+  Completed: (task) => task.completed,
+};
+
+const FILTER_NAMES = Object.keys(FILTER_MAP);
+
+export default class App extends Component {
+  state = {
+    newTodo: "",
+    createdDate: "",
+    completed: false,
+    filter: "All",
+    isOldestFirst: true,
+    sortBy: "addedDate",
+    sortAsc: true,
+    items: [],
+  };
+
+  componentDidMount() {
+    this.fetchTasks();
+  };
+
+  /**
+   * Function to get list of tasks
+   */
+  fetchTasks = async () => {
+    try {
+      const { data: response } = await http.get("/todoitems");
+      
+      this.setState({ items: response });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+// fetchTasks = async () => {
+//   try {
+//     const response = todoService.fetchTask();
+//     console.log("res", response);
+//   }
+//   catch (error) {
+//     console.log("error");
+//   }
+// };
+
+
+  /**
+   * Function to get value when new task add
+   */
+  handleOnChange = (event) => {
+    this.setState({ newTodo: event.target.value });
+  };
+
+  /**
+   * Function to get created data
+   * @param {*} date
+   */
+  handleDate = (date) => {
+    this.setState({
+      createdDate: `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`,
+    });
+  };
+
+  /**
+   * Function to create new task
+   * @param {*} event
+   */
+  handleOnSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const response = await http.post("/todoitems", {
```

Comment:
If we follow the best pratice, we would use this string 
" /todoitems " from here: 
![Uploading image.png…]()